guide/concepts/bins.md

```diff
@@ -0,0 +1,35 @@
+# Bins
+
+Bins, similarly to pipelines, are containers for elements. However, at the same time,
```

Comment:
Well, they can also be containers for other bin (recursively). I would insist on using proper terminology, element is no longer equivalent to child because a child can also be a bin. The same applies to being a parent.

Reply:
It applies to more places in this text.

Reply:
I agree about the other places, but at the beginning I'd stick to _elements_ not to overcomplicate things. The fact that bins can be nested is pointed out two sentences later.